README.md

```diff
@@ -140,6 +140,38 @@ This performs an NVE molecular dynamics simulation at 300K for 200 steps (0.2 ps
 For all options, run `janus md --help`.
 
 
+### Using configuration files
+
+Default values for all commandline options may be specifed through a Yaml 1.1 formatted configuration file by adding the `--config` option.
+
+For example, with the following configuration file and command:
+
+```yaml
+struct_path: "NaCl.cif"
+properties:
+  - "energy"
+out_file: "NaCl-results.xyz"
+arch: mace_mp
+calc_kwargs:
+  model: medium
+```
+
+```shell
+janus singlepoint --struct KCl.cif --out KCl-results.cif --config config.yml
+```
+
+This will run a singlepoint energy calculation on `KCl.cif` using the [MACE-MP](https://github.com/ACEsuit/mace-mp) "medium" force-field, saving the results to `KCl-results.cif`.
+
+
+> [!NOTE]
+> `properties` must be passed as a Yaml list, as above, not as a string.
+
+> [!WARNING]
+> Options in the Yaml file must match their Python variable name, which may differ from the option flag.
```

Comment:
this is not cool... we need to think.

Reply:
Yes it's not ideal. The obvious solution is just make them all match

Reply:
yap i think that is the solution otherwise too much confusion.

Reply:
There will still be a `-` vs `_` difference, but probably close enough otherwise.

It would be much less of an issue if it didn't allow unspecified options too, so I need to figure out if that's fixable too